app/Services/GitHubClientGapTracker.php

```
@@ -0,0 +1,468 @@
+            $pr = Github::pullRequests()->get($owner, $repo, $prNumber);
+            $headSha = $pr->head->sha ?? null;
+
+            if (!$headSha) {
+                $gaps['no_head_sha'] = 'Cannot get commit SHA for check status';
+                return $gaps;
+            }
+
+            // Test check runs endpoint
+            try {
+                if (method_exists(Github::class, 'checks')) {
+                    $checkRuns = Github::checks()->runs($owner, $repo, $headSha);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$checkRuns'. (Unused Code Rules)

(UnusedLocalVariable)